### PR TITLE
Adding services table to vmi details tab

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

## 📝 Description

Added services table to VMI details tab

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachinesinstance/details/tabs/details/components/Services/Services.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { ServicesList } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Title } from '@patternfly/react-core';
+import { LinkIcon } from '@patternfly/react-icons';
+
+const Services = ({ vmi, pathname }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [services, loaded] = useK8sWatchResource<IoK8sApiCoreV1Service[]>({
+    kind: ServiceModel?.kind,
```
Comment (avivtur) [inline src/views/virtualmachinesinstance/details/tabs/details/components/Services/Services.tsx:16]: kind is about to be depracated, use groupVersionKind: modelToGroupVersionKind(ServiceModel) instead

Comment (metalice) [reply src/views/virtualmachinesinstance/details/tabs/details/components/Services/Services.tsx]: great, changed.

**Pull request merged**